R/pkg/R/column.R

```diff
@@ -44,6 +44,9 @@ setMethod("initialize", "Column", function(.Object, jc) {
   .Object
 })
 
+#' @rdname column
+#' @name column
+#' @aliases column,jobj-method
```

Comment:
So for better or worse it is callable. i.e. the following code works
```
a <- createDataFrame(faithful)
b <- a$eruptions
c <- column(b@jc) // This is creating a column with a jobj
```
It might be confusing to users though -- thats something we have to live with I guess